docs/releases/release2100.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 99.85
+---
+
+# Release 2.10.0
+
+Date: 15 of April, 2025
+
+## Overview
+
+Klaw version 2.10.0 is a minor release with security updates and several improvement.
+With v2.10.0, users with the role `SUPERADMIN` will have access to the new React UI for certain screens.
+
+## Download
+
+### Binaries
+
+[//]: # "@TODO links"
+
+[`klaw-2.10.0.jar` ⬇︎](https://github.com/Aiven-Open/klaw/releases/download/v2.9.0/klaw-2.9.0.jar)
```

Comment:
```suggestion
[`klaw-2.10.0.jar` ⬇︎](https://github.com/Aiven-Open/klaw/releases/download/v2.10.0/klaw-2.10.0.jar)
```

Reply:
🤦‍♀️  I even had a TODO there. Thanks for catching this!